test/integration/utils.go

```diff
@@ -131,20 +131,6 @@ func GetEnvValueByName(envs []corev1.EnvVar, name string) string {
 	return value
 }
 
-// SetEnvValueByName sets the EnvVar in slice with the provided name and value.
-func SetEnvValueByName(envs []corev1.EnvVar, name string, value string) []corev1.EnvVar {
-	for _, env := range envs {
-		if env.Name == name {
-			env.Value = value
-			return envs
-		}
-	}
-	return append(envs, corev1.EnvVar{
-		Name:  name,
-		Value: value,
-	})
-}
```

Comment:
Maybe, worth reporting this case to [ineffassign](https://github.com/gordonklaus/ineffassign). WDYT?
We use it as part of `golanci-lint` and in the ideal world it should catch this

Reply:
🆒